### ISSUE TITLE
Tabs need to function as "links" to the content (for accessibility)

### ISSUE THREAD

**Issue by andrewlarkin** [Comcast/Xooie#6]
When clicking a tab the content area needs to gain focus.  Content area is made focusable by adding "tabindex=0" and an aria-labeled by tag should be added referencing the tab.
